auth/services/irma/irmaconfig.go

```diff
@@ -47,8 +48,27 @@ func GetIrmaConfig(validatorConfig ValidatorConfig) (irmaConfig *irma.Configurat
 		return
 	}
 
+	// This fixes an IRMA bug https://github.com/privacybydesign/irmago/issues/139
+	// It removes any temporary "tempscheme123" directories, leftover from a previous unfinished schema update.
+	dirs, err := filepath.Glob(filepath.Join(validatorConfig.IrmaConfigPath, "tempscheme*"))
+	if err != nil {
+		return
+	}
+	for _, dir := range dirs {
+		log.Logger().Infof("removing leftover temporary IRMA scheme dir: %s", dir)
+		if err := os.RemoveAll(dir); err != nil {
+			log.Logger().Warnf("unable to remove dir: %s", err)
```

Comment:
I think this is an error to be returned? Otherwise, the node can't start anyways?

Reply:
So, perhaps just ignore it?

Reply:
Return the error, I'd say

Reply:
I think this is bugfix code, if it fails, just try to continue, perhaps the failure is not critical, the actual execution of the lines down below will tell.